File: 23 Sept 2025/main.java
import java.io.*;
import java.util.*;

class Main{
    public static void main(String[] args){
        String[] students ={"Mg Mg","Aye Aye","Kyaw Kyaw"};
        try{
            FileWriter myFile = new FileWriter("student_list.txt");
            for(int i=0; i < students.length; i++){
                myFile.write(students[i]+",");
            }
            myFile.close();
        }catch(IOException e){
            System.out.println("Error:"+e);
        }
    }

}

File: 23 Sept 2025/main.py
# File Write
# with open('smk.txt','w') as myFile:
#     myFile.write('Student 1'+'\n')
#     myFile.write('Student 2')

# myFile = open('smk.txt','w')
# myFile.write('Student 3'+'\n')
# myFile.write('Student 4')
# myFile.close()

# File Read
with open('smk.txt','r') as myFile:
    student_list = myFile.read()
    print(student_list)

student_list=[]
myFile = open('smk.txt','r')
student_list = myFile.read().split(',')
print(student_list)
myFile.close()